cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -644,7 +736,11 @@ def horovod_spark_wf(
     # work_dir corresponds to the Horovod-Spark store
     return horovod_spark_task(data_dir=data_dir, hp=hp, work_dir=work_dir)
 
-
+# %%
+# Running the model locally
+# =============
+#
+# We can run the code locally too!
```

Comment:
```suggestion
# %%
# Running the Model Locally
# ====================
#
# We can run the code locally too, given Spark is enabled and the plugin is set up in the environment.
```

@SmritiSatyanV, can you include a link to the Spark tutorial in the README? I forgot to mention it.